flight_phase_files/HALO_RF01_20200119_info.yaml

```diff
@@ -26,15 +26,36 @@ remarks:
 - Clear sky during take off in Santiago de Compostela.
 - Super curtain from NTAS buoy over ships towards circle center.
 segments:
-- kind: radar_calibration_wiggle
-  name: radar calibration with +-20 deg roll wiggles
+- kinds:
+  - radar_calibration_wiggle
+  name: Radar calibration with +-20 deg roll wiggles
+  irregularities: []
+  segment_id: HALO-0119_rcw1
+  start: 2020-01-19 16:08:51
+  end: 2020-01-19 16:11:36
+- kinds:
+  - straight_leg
+  - super_curtain
+  name: Super curtain (from NTAS, over ships, to circle center)
+  irregularities: []
+  segment_id: HALO-0119_sl1
   start: 2020-01-19 16:55:16
   end: 2020-01-19 17:22:00
-- kind: straight_leg
-  name: super curtain (from NTAS, over ships, to circle center)
-  start: 2020-01-19 16:55:16
-  end: 2020-01-19 17:22:00
-- kind: circle
-  name: circle 1
-  start: 2020-01-19 17:32:27
-  end: 2020-01-19 18:22:23
+- kinds:
+  - circle
+  name: Circle 1
+  irregularities:
+  - Start of circle directly with first drop, roll angle around 4 deg in beginning
+  segment_id: HALO-0119_c1
+  start: 2020-01-19 17:33:26
+  end: 2020-01-19 18:24:13
+  good_dropsondes: 11
+- kinds:
+  - circling
+  name: Circling
+  irregularities:
+  - Start of circling directly with first drop
+  segment_id: HALO-0119_o1
+  start: 2020-01-19 17:33:26
+  end: 2020-01-19 18:25:31
```

Comment:
HALO continued circling a little longer than just for the 360 degree. Since people might also be interested in the times HALO was on the circular pattern, but not directly on a dropsonde circle, I extended the times to include all time steps on the circle.